packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -43,6 +44,12 @@ const NominationBanner = (props) => {
     ActiveNominationContext
   );
 
+  const [showDocsButton, setShowDocsButton] = useState(false);
+
+  const openWindow = (val) => {
+    window.open(`https://drive.google.com/drive/u/5/folders/${val}`);
```

Comment:
The `/u/5` part of this is specific to your browser setup. Would probably be best just to leave it out.

Reply:
fixed

Reply:
I'm not sure what you mean by "image you linked" @vhoof . The link goes to a google drive folder created by the api.